### DISCUSSION TITLE
R lessons!

### DISCUSSION THREAD

**Discussion by jlooper** [microsoft/ML-For-Beginners#249] [Announcements]
Because of our awesome Microsoft Student Ambassador Eric Wanjau's work, you can take some of the lessons in R! Take a look:

https://github.com/microsoft/ML-For-Beginners/tree/main/2-Regression/1-Tools/solution
and
https://github.com/microsoft/ML-For-Beginners/tree/main/2-Regression/2-Data/solution

thank you, Eric!

**Comment by jlooper** [comment]
@R-icntay and @ravindranath-sawane hey 'R' people - maybe you can work together here :)

**Reply by R-icntay** [reply]
Hey @ravindranath-sawane, happy to have you on board! Allow me to get back to this shortly on how we could collaborate to paRse the remaining lessons :)

**Reply by R-icntay** [reply]
Hello @ravindranath-sawane. I hope you are doing great. I was thinking of how we could collaborate to get the R versions of the lessons up and available. My initial thought is that I will take the Clustering and Natural language processing lessons and you take the Time series and Reinforcement learning lessons. A pair of lessons for both of us :).

How does that sound?

**Reply by ravindranath-sawane** [reply]
Hey, @R-icntay Sorry I would not be able to collaborate for R lessons.. My PC is not supporting R-Studio there is some problem.. I hope you'll understand.

**Comment by jlooper** [comment]
@R-icntay - could you do a spelling audit of the classification lesson - I found a reference to 'tibble' a few times - should it be 'table'?

**Reply by R-icntay** [reply]
Hello Jen,

So in R, a tibble is kind of a modern re-imaging of the data frame. 

Probably mentioned it in the previous lessons but will reiterate that in the classification lesson (Or just call them data frames 😄)

**Reply by jlooper** [reply]
I had a feeling there was something R-ish that I didn't understand! Maybe you can switch them back... sorry for my ignorance!

**Reply by R-icntay** [reply]
Don't you worry Jen,

Us R folks are quite an understanding and welcoming bunch of people.

Since tibble is actually a package in the Tidyverse, I guess this is the part we say:
Welcome to the Tidyverse! :)

**Comment by chickpink22** [comment]
Heyy

**Reply by R-icntay** [reply]
Hello!

**Comment by magothe-cloud** [comment]
hello

**Reply by R-icntay** [reply]
Hello @magothe-cloud ...
